components/nimbus/metrics.yaml

```diff
@@ -103,7 +160,7 @@ nimbus_events:
       branch:
         type: string
         description: The branch slug/identifier that was randomly chosen
-      enrollment_id:
+      feature_id:
         type: string
         description: A unique identifier generated at enrollment time
```

Comment:
Is this still a good descriptor for `feature_id`?

Reply:
Nope, good catch!